Classes/Service/SettingsService.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Skyfillers\SfSimpleFaq\Service;
+namespace SKYFILLERS\SfSimpleFaq\Service;
```

Comment:
I changed it to uppercase. In the old file it was lowercase and i got an error due to that.